### ISSUE TITLE
The python demo_alter.py hangs on: "Input video number of frames 300"

### ISSUE THREAD

**Issue by dangerstudios** [carlosedubarreto/vibe_win_install#9]
so everything installed ok but now the demo hangs here(rtx 3090):

```
(venv_vibe2) D:\vibe\VIBE>python demo_alter.py --vid_file sample_video.mp4 --output_folder output/ --display
Running "ffmpeg -i sample_video.mp4 -f image2 -v error /tmp\sample_video_mp4/%06d.png"
Images saved to "/tmp\sample_video_mp4"
Input video number of frames 300
```

**Comment by carlosedubarreto** [comment]
Sorry but this kind of problem I dont have experience.
Search for help on the Vibe page, you probably will have a better chance to fix the issue.

First search for an issue that can be similar to the problem you are having and if you dont find, open a new issue on this page.
https://github.com/mkocabas/VIBE/issues

**Comment by goldensun2333** [comment]
I have the same problem as @dangerstudios, and my GPU is RTX 3080ti. The problem shows the program run on CPU even if the CUDA is installed and recognized successfully. It seems like the Nvidia 30 series GPUs only support CUDA 11.0 or above version. Could you modify the package requirement to support 30 series GPU? Thanks!

**Comment by carlosedubarreto** [comment]
Hi @goldensun2333, I dont have the knowledge to do that,  neither how to test.
The only thing I can do is give the  direction to where you could find something or ask where there is more capable people to help you on that, that is going to the real repository.

Here is just a place to try and help on windows installation for cuda 10 cards

https://github.com/mkocabas/VIBE/issues

**Comment by tscftw** [comment]
Yeah I also have this issue, I tried installing on linux as well but the same problems happened there (although i did end up copying files from my windows install due to other problems) I'm also running a 3080 so it might just be 30 series cards CUDA version issue

**Comment by carlosedubarreto** [comment]
if you are installing on linux, it should runs smoother using the official repository instructions, and this repository is useless., since its a try to make it run on windows.

**Comment by tscftw** [comment]
I installed on windows first, then tried on linux and had to use the windows files to fix some issues, I actually ended up getting it working on linux for a bit (still on cpu but it was quick enough) but haven't been able to replicate it😕? I think im just going to install [PARE](https://github.com/mkocabas/PARE) instead as it seems to be the successor to this project.

**Comment by carlosedubarreto** [comment]
might be a better alternative, and thanks for the link.
I didnt know they released the code (the last time I saw it was not available)
